plugins/gatsby-plugin-generate-doc-json/gatsby-node.js

```diff
@@ -29,39 +29,40 @@ const htmlGenerator = unified()
 exports.onPostBuild = async ({ graphql, store }) => {
   const { program } = store.getState();
 
-  const query = `query {
-    allMdx(filter: {fileAbsolutePath: {regex: "/src/content/docs/"}}) {
-      nodes {
-        mdxAST
-        slug
-        fields {
-          fileRelativePath
+  try {
+    const { data } = await graphql(`
+      query {
+        allMdx(filter: { fileAbsolutePath: { regex: "/src/content/docs/" } }) {
+          nodes {
+            mdxAST
+            slug
+            fields {
+              fileRelativePath
+            }
+          }
         }
-      }
-    }
-    allFile(filter: {childImageSharp: {id: {ne: null}}}) {
-      nodes {
-        childImageSharp {
-          fluid {
-            src
+        allImageSharp {
+          nodes {
+            parent {
+              ... on File {
+                relativePath
+              }
+            }
+            fluid {
+              src
+            }
           }
         }
-        relativePath
       }
-    }
-  }`;
+    `);
 
-  try {
-    const { data } = await graphql(query);
-    const { allMdx, allFile } = data;
+    const { allMdx, allImageSharp } = data;
 
-    const imageHashMap = allFile.nodes.reduce(
-      (obj, { childImageSharp, relativePath }) => {
-        return {
-          ...obj,
-          [relativePath]: childImageSharp.fluid.src,
-        };
-      },
+    const imageHashMap = allImageSharp.nodes.reduce(
+      (memo, { fluid, parent }) => ({
```

Comment:
is there a reason for call this memo? It's not really a memoized function right?

Reply:
Ah this harkens back to my Ruby days where this variable was called `memo` as a generic term for the accumulator (https://ruby-doc.org/core-2.4.0/Enumerable.html#method-i-reduce). I forget that in JS its usually called "accumulator" as the shorthand. Thats the only reason. This wasn't meant to mean "memoized value", my brain just went to ruby-land generic term.

Reply:
ah that makes sense! would it be possible to change it to `acc`? Just because in my brain, memo always means some sort of memoized value or function (like `useMemo`)

Reply:
Sure!